src/python/tests/ops_simnoise.py

```diff
@@ -1,25 +1,17 @@
 # Copyright (c) 2015-2017 by the parties listed in the AUTHORS file.
-# All rights reserved.  Use of this source code is governed by 
+# All rights reserved.  Use of this source code is governed by
 # a BSD-style license that can be found in the LICENSE file.
 
-from ..mpi import MPI
-from .mpi import MPITestCase
-
-import sys
 import os
 
 import numpy as np
-import numpy.testing as nt
-
-import scipy.interpolate as si
 
-from ..tod.tod import *
-from ..tod.pointing import *
-from ..tod.noise import *
-from ..tod.sim_noise import *
-from ..tod.sim_det_noise import *
-from ..tod.sim_tod import *
```

Comment:
The wildcard import statements caused much grief with pylint. I have also grouped them according to the style guide to pass the pylint checks.